components/game/Tetris.tsx

```diff
@@ -18,13 +18,15 @@ const Tetris = () => {
   const [heldField, setHeldField] = useState<number[][]>(
     TetrisConsts.RENDER_TETROMINOS_ARR[0]
   );
-  const [spawnedFields, setSpawnedFields] = useState<JSX.Element[]>([]);
+  const [spawnedFields, setSpawnedFields] = useState<JSX.Element[]>(
+    Array(4).fill(<div className="tetris-next" />)
```

Comment:
Minor nitpick here but I think we should declare a constant for the magic number `4` for better readability :thinking:

Reply:
Replaced with a constant!